src/daml/Daml/Trading/Cash/Util.daml

```diff
@@ -0,0 +1,8 @@
+module Daml.Trading.Cash.Util where
+
+-- | Fetches a contract, archives it and returns its value.
+fetchAndArchive : Template a => ContractId a -> Update a
```

Comment:
please remove the file if it is not used anywhere

Reply:
Done